.github/workflows/auto-author-assign.yml

```diff
@@ -6,13 +6,14 @@ on:
 
 env:
   IS_FORK: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
+  IS_DEPENDABOT: ${{ github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]' }}
```

Comment:
nit: can we introduce new var JOB_ALLOWED = IS_FORK == false && IS_DEPENDABOT == false(or something similar), and use it below?

Reply:
good point)